modules/camera/camera_picamera2.py

```diff
@@ -3,6 +3,7 @@
 """
 
 import numpy as np
+from .camera_config import PiCameraConfig
```

Comment:
Please import modules, that's sort of a style thing to be consistent: `from . import camera_config`